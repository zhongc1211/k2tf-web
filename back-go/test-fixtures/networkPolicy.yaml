---
apiVersion: networking.k8s.io/v1
kind: NetworkPolicy
metadata:
  name: default-deny
spec:
  podSelector:
    matchLabels:
      IngressInternal: active
  ingress:
      - from:
        - ipBlock:
            cidr: 10.0.0.0/8
        - ipBlock:
            cidr: 172.16.0.0/12
        - ipBlock:
            cidr: 192.168.0.0/16
        ports:
          - port: 80
            protocol: TCP
          - port: 443
            protocol: TCP
  policyTypes:
    - Ingress
    - Egress
