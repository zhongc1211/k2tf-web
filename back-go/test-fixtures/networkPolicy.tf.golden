resource "kubernetes_network_policy" "default_deny" {
  metadata {
    name = "default-deny"
  }
  spec {
    pod_selector {
      match_labels = { IngressInternal = "active" }
    }
    ingress {
      ports {
        protocol = "TCP"
        port     = "80"
      }
      ports {
        protocol = "TCP"
        port     = "443"
      }
      from {
        ip_block {
          cidr = "10.0.0.0/8"
        }
      }
      from {
        ip_block {
          cidr = "172.16.0.0/12"
        }
      }
      from {
        ip_block {
          cidr = "192.168.0.0/16"
        }
      }
    }
    policy_types = ["Ingress", "Egress"]
  }
}
